samples/notebooks/azure_submission.ipynb

```diff
@@ -1,240 +1,168 @@
-            "text/plain": []
-          },
-          "metadata": {},
-          "output_type": "display_data"
-        },
-        {
-          "data": {
-            "application/x.qsharp-config": "{\"targetProfile\":\"base\",\"languageFeatures\":null,\"manifest\":null}",
-            "text/plain": [
-              "Q# initialized with configuration: {'targetProfile': 'base', 'languageFeatures': None, 'manifest': None}"
-            ]
-          },
-          "execution_count": 1,
-          "metadata": {},
-          "output_type": "execute_result"
-        }
-      ],
-      "source": [
-        "import qsharp\n",
-        "\n",
-        "qsharp.init(target_profile=qsharp.TargetProfile.Base)\n"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {},
-      "source": [
-        "The following will generate a compiler error because it uses a feature unsupported by the base profile."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": 2,
-      "metadata": {
-        "vscode": {
-          "languageId": "qsharp"
-        }
-      },
-      "outputs": [
-        {
-          "ename": "QSharpCellError",
-          "evalue": "\u001b[31mQsc.CapabilitiesCk.UseOfDynamicBool\u001b[0m\n\n  \u001b[31m×\u001b[0m cannot use a dynamic bool value\n   ╭─[\u001b[36;1;4mline_0\u001b[0m:4:1]\n \u001b[2m4\u001b[0m │     let result = M(q);\n \u001b[2m5\u001b[0m │     if (result == Zero) {\n   · \u001b[35;1m        ──────────────\u001b[0m\n \u001b[2m6\u001b[0m │         Message(\"The result is Zero\");\n   ╰────\n\u001b[36m  help: \u001b[0musing a bool value that depends on a measurement result is not\n        supported by the current target\n",
-          "output_type": "error",
-          "traceback": [
-            "\u001b[31mQsc.CapabilitiesCk.UseOfDynamicBool\u001b[0m",
-            "",
-            "  \u001b[31m×\u001b[0m cannot use a dynamic bool value",
```

Comment:
Looks like some of these checked-in outputs were removed, but per the input cell is supposed to show the error.

Reply:
@minestarks mentioned this was intentional, so ignore this comment. Thanks!